src/gui/generalsettings.ui

```diff
@@ -269,6 +225,61 @@
         </item>
        </layout>
       </item>
+      <item>
+       <widget class="QWidget" name="horizontalLayoutWidget_remotePollInterval">
+        <layout class="QHBoxLayout" name="horizontalLayout_remotePollInterval">
+         <item>
+          <widget class="QLabel" name="remotePollIntervalLabel">
+           <property name="text">
+            <string>Server poll interval</string>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QSpinBox" name="remotePollIntervalSpinBox">
+           <property name="minimum">
+            <number>30</number>
+           </property>
+           <property name="maximum">
+            <number>999999</number>
+           </property>
+           <property name="singleStep">
+            <number>1</number>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QLabel" name="labelInterval">
+           <property name="text">
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;seconds (if &lt;a href=&quot;https://github.com/nextcloud/notify_push&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Client Push&lt;/span&gt;&lt;/a&gt; is unavailable)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+           </property>
+           <property name="textFormat">
+            <enum>Qt::TextFormat::RichText</enum>
+           </property>
+           <property name="openExternalLinks">
+            <bool>true</bool>
+           </property>
+           <property name="textInteractionFlags">
+            <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
```

Comment:
the [`TextBrowserInteraction` flag also sets `Qt::TextSelectableByMouse`](https://doc.qt.io/qt-6/qt.html#TextInteractionFlag-enum), which is not really wanted on most labels.  It's enough for the links to be accessible by keyboard and mouse only.

```suggestion
            <set>Qt::TextInteractionFlag::LinksAccessibleByKeyboard|Qt::TextInteractionFlag::LinksAccessibleByMouse</set>
```